androidbrowserhelper/src/main/java/com/google/androidbrowserhelper/trusted/TwaLauncher.java

```diff
@@ -219,6 +224,20 @@ public void destroy() {
         mDestroyed = true;
     }
 
+    public static final FallbackStrategy cctFalbackStrategy =
+            (context, twaBuilder, providerPackage, completionCallback) -> {
+        // CustomTabsIntent will fall back to launching the Browser if there are no Custom Tabs
+        // providers installed.
+        CustomTabsIntent intent = twaBuilder.buildCustomTabsIntent();
+        if (providerPackage != null) {
+            intent.intent.setPackage(providerPackage);
+        }
+        intent.launchUrl(context, twaBuilder.getUrl());
+        if (completionCallback != null) {
+            completionCallback.run();
+        }
```

Comment:
I think we should move this part out of the fallbackStrategy - we don't want users to forget about calling it.

So take a FallbackStrategy (which is now only a `(context, twaBuilder, providerPackage) -> ...`) and then maybe within launch we wrap it in another callback that calls it and then completionCallback.